src/api/githubApi.ts

```diff
@@ -0,0 +1,35 @@
+// src/api/githubApi.ts
+let githubToken = '';
+
+export const saveToken = (token: string) => {
+  githubToken = token;
+  localStorage.setItem('github_token', token); // 保存 token 到 localStorage
+};
+
+export const getToken = () => {
+  if (!githubToken) {
+    githubToken = localStorage.getItem('github_token') || '';
+  }
+  return githubToken;
+};
```

Comment:
save and get token are not api related, could you move them to `helpers/`?

Reply:
I think I meant to move only the saveToken and getToken to helper and keep the githubRequest in the api folder lol